app/scripts/modules/core/src/pipeline/config/triggers/triggers.directive.js

```diff
@@ -33,8 +33,14 @@ module.exports = angular.module('spinnaker.core.pipeline.config.trigger.triggers
       $scope.pipeline.triggers.push(newTrigger);
     };
 
+    this.defaultArtifact = () => {
```

Comment:
I like to omit `return` on these kinds of object factories, but not for any solid reason: 
```
this.defaultArtifact = () => ({
  kind: 'custom',
});
```

Reply:
Looks cleaner